app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+   if logged_in?
+     @tweets = Tweet.all
+     erb :'/tweets/tweets'
+   else
+     redirect to '/login'
+   end
+ end
+
+ get '/tweets/new' do
+   if !logged_in?
+     redirect to '/login'
+   else
+     erb :'/tweets/create_tweet'
+   end
+  end
+
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+      @tweet = Tweet.create(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```